ldap-api/src/main/java/ca/bc/gov/hlth/ldapapi/service/LdapService.java

```diff
@@ -106,37 +106,10 @@ private boolean checkUserLocked(Attributes attributes) {
         return unlocked;
     }
 
-    private boolean checkUserPasswordExpired(Attributes attributes) {
-        Attribute passwordChangeDate = attributes.get(LDAP_ATTR_PASSWORD_CHANGE_DATE);
-        Attribute passwordLifespan = attributes.get(LDAP_ATTR_PASSLIFESPAN);
-        boolean expired = false;
-
-        if (passwordChangeDate != null) {
-            try {
-                long epochDateLastChangedOn = Long.parseLong(passwordChangeDate.get().toString());
```

Comment:
A blank "Password Lifespan" causes this to throw an exception, but we don't need the password expiry value anyhow so the easiest fix is just to remove the functionality.